xmodule/modulestore/tests/test_semantics.py

```diff
@@ -446,9 +446,18 @@ def test_update_asides(self, block_type):
                                    self.ASIDE_DATA_FIELD.field_name, self.ASIDE_DATA_FIELD.updated)
 
 
+@ddt.ddt
 class TestMongoDirectOnlyCategorySemantics(DirectOnlyCategorySemantics):
     """
     Verify DIRECT_ONLY_CATEGORY semantics against the MongoModulestore
     """
     MODULESTORE = TEST_DATA_MONGO_MODULESTORE
     __test__ = True
+
+    @ddt.data(ModuleStoreEnum.Branch.draft_preferred, ModuleStoreEnum.Branch.published_only)
+    def test_course_summaries(self, branch):
+        """ Test that `get_course_summaries` method in modulestore work as expected. """
+        with self.store.branch_setting(branch_setting=branch):
+            course_summaries = self.store.get_course_summaries()
+            # get_course_summaries skip old mongo courses
+            assert len(course_summaries) == 0
```

Comment:
The course summaries should still be generated for Old Mongo courses, shouldn't they? This was the basic metadata like display_name?

Reply:
Oh yes you are right, course summaries should still be generated for Old Mongo courses. I'll update it.

Reply:
updated